src/main/java/frc/robot/Robot.java

```diff
@@ -57,6 +119,86 @@ public void robotInit() {
+            /* for advanced users.  Before using this data, please consider whether     */
+            /* the processed data (see above) will suit your needs.                     */
+            
+            SmartDashboard.putNumber(   "RawGyro_X",            ahrs.getRawGyroX());
+            SmartDashboard.putNumber(   "RawGyro_Y",            ahrs.getRawGyroY());
+            SmartDashboard.putNumber(   "RawGyro_Z",            ahrs.getRawGyroZ());
+            SmartDashboard.putNumber(   "RawAccel_X",           ahrs.getRawAccelX());
+            SmartDashboard.putNumber(   "RawAccel_Y",           ahrs.getRawAccelY());
+            SmartDashboard.putNumber(   "RawAccel_Z",           ahrs.getRawAccelZ());
+            SmartDashboard.putNumber(   "RawMag_X",             ahrs.getRawMagX());
+            SmartDashboard.putNumber(   "RawMag_Y",             ahrs.getRawMagY());
+            SmartDashboard.putNumber(   "RawMag_Z",             ahrs.getRawMagZ());
+            SmartDashboard.putNumber(   "IMU_Temp_C",           ahrs.getTempC());
+            SmartDashboard.putNumber(   "IMU_Timestamp",        ahrs.getLastSensorTimestamp());
+            
+            /* Omnimount Yaw Axis Information                                           */
+            /* For more info, see http://navx-mxp.kauailabs.com/installation/omnimount  */
+            AHRS.BoardYawAxis yaw_axis = ahrs.getBoardYawAxis();
+            SmartDashboard.putString(   "YawAxisDirection",     yaw_axis.up ? "Up" : "Down" );
+            SmartDashboard.putNumber(   "YawAxis",              yaw_axis.board_axis.getValue() );
+            
+            /* Sensor Board Information                                                 */
+            SmartDashboard.putString(   "FirmwareVersion",      ahrs.getFirmwareVersion());
+            
+            /* Quaternion Data                                                          */
+            /* Quaternions are fascinating, and are the most compact representation of  */
+            /* orientation data.  All of the Yaw, Pitch and Roll Values can be derived  */
+            /* from the Quaternions.  If interested in motion processing, knowledge of  */
+            /* Quaternions is highly recommended.                                       */
+            SmartDashboard.putNumber(   "QuaternionW",          ahrs.getQuaternionW());
+            SmartDashboard.putNumber(   "QuaternionX",          ahrs.getQuaternionX());
+            SmartDashboard.putNumber(   "QuaternionY",          ahrs.getQuaternionY());
+            SmartDashboard.putNumber(   "QuaternionZ",          ahrs.getQuaternionZ());
+            
+            /* Connectivity Debugging Support                                           */
+            SmartDashboard.putNumber(   "IMU_Byte_Count",       ahrs.getByteCount());
+            SmartDashboard.putNumber(   "IMU_Update_Count",     ahrs.getUpdateCount());
+		Cameras.run();
```

Comment:
This is a **lot** of stuff to be printing to the console. Might make it hard to see important stuff on the console